ui/app/AppLayouts/Wallet/addaccount/panels/DerivationPath.qml

```diff
@@ -103,13 +100,11 @@ GridLayout {
 
                 onSelected: {
                     root.store.changeRootDerivationPath(rootPath)
+                    derivationPathInput.resetDerivationPath(root.store.selectedRootPath, root.store.addAccountModule.derivationPath)
                 }
             }
-        }
 
-        Connections {
```

Comment:
Removing this update of the base path every time it changes in the backend fixes the problem. As we talked about before, this was also the workaround I wanted to aply, but I hoped you could fix the root problem of why does it change in the store.

Also, this is not needed at all to move the declarative initialization implementation (from the `DerivationPathInput` control) into an imperative one. However, if you prefer an imperative way of initialization, then the storybook must also be updated.